packages/dev/loaders/src/SPLAT/splatFileLoader.ts

```diff
@@ -43,6 +44,24 @@ interface ParsedPLY {
     hasVertexColors?: boolean;
 }
 
+/** @internal */
+class CompressedPLYChunk {

```

Comment:
instead of a class this can simple be an interface and a new object can be created implementing this interface. No need for the overhead of class creation